soh/soh/Enhancements/boss-rush/BossRush.cpp

```diff
@@ -632,6 +632,10 @@ void BossRush_OnVanillaBehaviorHandler(GIVanillaBehavior id, bool* should, void*
             *should = false;
             break;
         }
+        // Prevent warning spam
+        default: {
```

Comment:
with c++ case statements, you don't always need {}'s. They are only required if you need to declare variables in a case's "scope", as the {}'s set a scope for the case to use.

Reply:
I know but I think it looks cleaner & more consistant if either all of them have it or none do.
As a plus it allows you to collapse it in vscode.